### ISSUE TITLE
newlines_after_classes fails when class ends with a function.

### ISSUE THREAD

**Issue by jopdeklein** [clutchski/coffeelint#230]
Considering the following config:
```javascript
{
    "newlines_after_classes": {
        "name": "newlines_after_classes",
        "value": 2,
        "level": "error"
    }
}
```

The newlines_after_classes rule works as expected when the class ends with a property:
```coffeescript
class Foo
  bar: "foo"

return Foo
```

But fails when the class ends with a function:
```coffeescript
class Foo
  bar: ->
    "foo"

return Foo
```

I added a failing test case to illustrate the issue:
https://github.com/jopdeklein/coffeelint/commit/e5c38bbf0ddf33c95aeac8ebfcb00fb293938494

**Comment by AsaAyers** [comment]
@xdissent Did you ever make your PR for this? Is this still broken? (I don't use this rule)
